docs/user/tutorial.rst

```diff
@@ -85,98 +93,164 @@ Note the method signature. ``env`` and ``start_response`` are standard
+user-friendly alternative. Let's install HTTPie and use it from now on:
 
 .. code:: bash
 
-    $ pip install --upgrade httpie
+    $ source .venv/bin/activate
+    $ pip install httpie
     $ http localhost:8000
 
-
 Creating Resources
 ------------------
 
-Falcon borrows some of its terminology from the REST architectural
-style, so if you are familiar with that mindset, Falcon should be familiar.
-On the other hand, if you have no idea what REST is, no worries; Falcon
-was designed to be as intuitive as possible for anyone who understands
-the basics of HTTP.
+Falcon's design borrows several key concepts from the REST architectural
+style.
+
+Central to both REST and the Falcon framework is the concept of a
+"resource". Resources are simply all the "things" in your API or
+application that can be accessed by a URL. For example, an event booking
+application may have resources such as "ticket" and "venue", while a
+video game backend may have resources such as "achievements" and
+"player".
+
+URLs provide a way for the client to uniquely identify resources. For
+example, ``/players`` might identify the "list of all players" resource,
+while ``/players/45301f54`` might identify the "individual player with
+ID 45301f54", and ``/players/45301f54/achievements`` the
+"list of all achievements for the player resource with ID 45301f54".
+
+.. code::
+
+      POST        /players/45301f54/achievements
+    └──────┘    └────────────────────────────────┘
+     Action            Resource Identifier
+
+In the REST architectural style, the URL only
+identifies the resource; it does not specify what action to take on
+that resource. Instead, users choose from a set of standard methods.
+For HTTP, these are the familiar GET, POST, HEAD, etc. Clients can
+query a resource to discover which methods it supports.
+
+.. note::
+
+    This is one of the key differences between the REST and RPC
+    architectural styles. REST applies a standard set of
+    verbs across any number of resources, as opposed to
+    having each application define its own unique set of methods.
+
+Depending on the requested action, the server may or may not return a
+representation to the client. Representations may be encoded in
+any one of a number of Internet media types, such as JSON and HTML.
 
-In Falcon, you map incoming requests to things called "Resources". A
-Resource is just a regular Python class that includes some methods that
-follow a certain naming convention. Each of these methods corresponds to
-an action that the API client can request be performed in order to fetch
-or transform the resource in question.
+Falcon uses Python classes to represent resources. These classes
+act as controllers in your application, converting incoming requests
+into a series of internal actions to be passed off to other components
+or backend services, collecting the results, and then finally using
+those results to compose a response back to the client.
```

Comment:
Good catch! I attempted to simplify this paragraph. Let me know how the new version reads.